internal_transfers/actions/jest.config.js

```diff
@@ -3,5 +3,4 @@ module.exports = {
   testEnvironment: "node",
   testRegex: "/tests/.*\\.(test|spec)?\\.(ts|tsx)$",
   moduleFileExtensions: ["ts", "tsx", "js", "jsx", "json", "node"],
-  resolveJsonModule: true,
```

Comment:
Sure you want to remove this?

Reply:
Definitely. It causes an error at compile time.